gbif/gbif_name_match/gbif_species_name_match.R

```diff
@@ -44,7 +50,7 @@ request_species_information <- function(df, name_col,
     # test incoming arguments
     assert_that(is.data.frame(df))
     assert_colnames(df, name_col, only_colnames = FALSE) # colname exists in df
-    gbif_terms <- match.arg(gbif_terms)
+    gbif_terms <- match.arg(gbif_terms, API_terms, several.ok = TRUE)
```

Comment:
`match.arg(gbif_terms, several.ok = TRUE)` after updating `gibf_terms` to the content of `API_terms`

Reply:
So, splitting this up in two checks? What would be the advantage?